### PR TITLE
refs #12152: add ruby 2.2.2 to smart proxy test matrix

### PR DESCRIPTION
**Pull request by witlessbird**

### CODE REVIEW

**Comment by domcleal** [comment]
You'll also need to update rvm on the slaves to make 2.2 available.  Do that under puppet/modules/slave/manifests/init.pp.

**Comment by domcleal** [comment]
(And preferably 2.2.3, since it's out.)

File: puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_develop.yaml [hunk 1]
```diff
@@ -23,6 +23,7 @@
           - 1.9.3
           - 2.0.0
           - 2.1
+          - 2.2.2
```
Comment (domcleal) [inline puppet/modules/jenkins_job_builder/files/theforeman.org/yaml/jobs/test_proxy_develop.yaml]: We define aliases in slave/manifests/init.pp, so you should be able to use "2.2" to mean whatever 2.2 release we aliased (currently 2.2.1).

**Comment by witlessbird** [comment]
@domcleal: updated 2.2.2 alias for slaves.

**Comment by domcleal** [comment]
Did you want to try 2.2.3 while we're here, if it works for you with rvm stable?

**Comment by witlessbird** [comment]
lemme try installing 2.2.3.

**Comment by witlessbird** [comment]
@domcleal: switched to 2.2.3

**Comment by domcleal** [comment]
Thanks @witlessbird, I'll check that it installs correctly.

**Pull request merged**